python/cugraph/cugraph/dask/link_analysis/hits.py

```diff
@@ -154,9 +154,9 @@ def hits(input_graph, tol=1.0e-5, max_iter=100,  nstart=None, normalized=True):
 
     client = default_client()
 
-    # FIXME Still compute renumbering at this layer in case str
-    # vertex ID are passed
-    input_graph.compute_renumber_edge_list(transposed=False)
+    # FIXME: 'legacy_renum_only' will not trigger the C++ renumbering
```

Comment:
Can you explain the FIXME?  Is the idea that we don't want to have to use this arg in the future?